library/swift/KeyValueStore.swift

```diff
@@ -0,0 +1,31 @@
+@_implementationOnly import EnvoyEngine
+import Foundation
+
+/// `KeyValueStore` is an interface that may be implemented to provide access to an arbitrary
```

Comment:
Can you document the docstrings for these two types separately so that only the user-facing one is included in the docstrings for `KeyValueStore`?

Also please document the functions.

Reply:
Sure thing.